src/cozy/wrappers/react-wrapper.jsx

```diff
@@ -71,18 +74,22 @@ const ReactWrapper = ({
         }
       >
         <CozyProvider client={client}>
-          <VaultProvider instance={client.getStackClient().uri} vaultData={vaultData}>
-            <BitwardenSettingsContext.Provider value={bitwardenData}>
-              <WebviewIntentProvider>
-                <BreakpointsProvider>
-                  <CozyTheme>
-                    <HashRouter>{props.children}</HashRouter>
-                    <Alerter />
-                  </CozyTheme>
-                </BreakpointsProvider>
-              </WebviewIntentProvider>
-            </BitwardenSettingsContext.Provider>
-          </VaultProvider>
+          <BarProvider searchOptions={{ enabled: false }}>
+            <DataProxyProvider>
```

Comment:
Does it mean we load the DataProxy iframe? Is it mandatory even if we don't use it?

Reply:
You are right it is not mandatory if search is disabled, I removed it 👍

Reply:
Maybe we will add search later, in this case I will add again the provider.